### PR TITLE
Remove unused iIpAddr member of CChannelInfo class

### PR DESCRIPTION
**Pull request by wferi**
Since #316 (Avoid showing IP address if no name in the musician
profile is given) was fully fixed by e7621af7, this field has always
been zero, bringing nothing but uncertainty.

### CODE REVIEW

File: src/server.cpp [hunk 1]
```diff
@@ -1413,7 +1413,6 @@ CVector<CChannelInfo> CServer::CreateChannelList()
             // append channel ID, IP address and channel name to storing vectors
```
Comment (pljones) [inline src/server.cpp:1413]: "IP Address"?

File: src/testbench.h [hunk 2]
```diff
@@ -143,7 +143,6 @@ public slots:
 
         case 4: // PROTMESSID_CONN_CLIENTS_LIST
             vecChanInfo[0].iChanID = GenRandomIntInRange ( -2, 20 );
-            vecChanInfo[0].iIpAddr = GenRandomIPv4Address().toIPv4Address();
```
Comment (pljones) [inline src/testbench.h:146]: I _think_ I'd like these to have explicit zero put in, in place of the ipAddr.  (Same at old 281 below.)

Comment (wferi) [reply src/testbench.h:146]: Explicit zero put into where? The point of this change is the removal of the `iIpAddr` class member.

File: src/server.cpp (hunk 1 above)
Comment (wferi) [reply src/server.cpp]: Thanks for the review, you've certainly got a point here. I pushed a new commit removing this comment altogether.

File: src/testbench.h (hunk 2 above)
Comment (pljones) [reply src/testbench.h:146]: Sorry, code overload...

**Review by softins — APPROVED** [review]

**Review by pljones — APPROVED** [review]

**Comment by ann0see** [comment]
Ready to merge?

**Comment by pljones** [comment]
Looks like someone will need to manually rebase.

```
# on the fork with origin as wferi/jamulus
git checkout master
git pull
git pull upstream master ;# where upstream is jamulussoftware/jamulus
git push
git checkout noipaddr
git rebase master
# git status, read the instructions, ask for help if needed, resolve conflicts
git push --force
```
That should bring the branch up to date.

**Comment by hoffie** [comment]
Thanks!

(No conflicts after @ann0see's merge of master; squash-merged to keep history clean)

**Comment by ann0see** [comment]
@hoffie So

> Looks like someone will need to manually rebase.

Was not needed? Didn't check it though

**Comment by hoffie** [comment]
> @hoffie So
> 
> > Looks like someone will need to manually rebase.
> 
> Was not needed? Didn't check it though

There were no conflicts to solve, so yes, not needed, as far as I can see.

**Pull request merged**